ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml

```diff
@@ -39,101 +39,55 @@ resources:
       memory: 300Mi
 tests:
 - always_run: false
-  as: pr-rosa-hcp
-  optional: true
-  steps:
-    cluster_profile: aws-sd-qe
-    test:
-    - ref: rosa-test-e2e-pull-request
-    workflow: rosa-aws-sts-hcp
-- always_run: false
-  as: pr-rosa-hcp-byo-kms-oidc-auditlog
```

Comment:
For there are no default test profiles for rosa classic(sts/non-sts) and HCP, let us keep these jobs as `always:false and optional:true`, and just update the test profiles with the complex configurations. It could provide more choices and reduce cost.

Reply:
We already have. The default jobs will be `rosa-hcp-advanced` and `rosa-advanced` . It's meaningless to keep the old job since we don't maintain them soon. And I would like the pre-submit jobs are exactly what we covered in the testing.

Reply:
The `rosa-hcp-advanced` and `rosa-advanced` are too large to be the default jobs. I want some basic workflows that do not need the specified features. For example, I regularly wish for a ROSA HCP cluster without any unnecessary options. This kind of cluster should not be broken by any feature enhancements.

Reply:
New profiles can be added if the two profiles cannot run stable. It has the most coverage for features. But for old jobs they are not useful anymore. Some cases are running based on profile settings.